apis/project.cattle.io/v3/schema/registry_credential.go

```diff
@@ -18,17 +19,35 @@ func (e RegistryCredentialMapper) ToInternal(data map[string]interface{}) error
 		return nil
 	}
 
-	auth := convert.ToString(data["auth"])
-	username := convert.ToString(data["username"])
-	password := convert.ToString(data["password"])
-
-	if auth == "" && username != "" && password != "" {
-		data["auth"] = base64.StdEncoding.EncodeToString([]byte(username + ":" + password))
+	if data["kind"] != "dockerCredential" {
```

Comment:
can this mapper be added only on dockerCredential in schema?

Reply:
It was not getting invoked for v3.DockerCredential